repos/c2sm/packages/oasis/package.py

```diff
@@ -21,7 +21,7 @@ class Oasis(MakefilePackage):
     git = 'https://gitlab.com/cerfacs/oasis3-mct.git'
     maintainers = ['leclairm']
 
-    version('master', branch='OASIS3-MCT_4.0')
+    version('OASIS3-MCT_4.0', tag='OASIS3-MCT_4.0')
```

Comment:
Are you sure this tag exists? I can't see it: https://gitlab.com/cerfacs/oasis3-mct/-/tags

Reply:
`'OASIS3-MCT_4.0` is a branch afaik.

Reply:
Damn it, yes it's a branch. they don't provide tags on this branch. Should I put a commit hash in the spack package instead?

Reply:
I don't know if you can. But if spack supports this, yes.

Reply:
> I don't know if you can. But if spack supports this, yes.

Yes, it does : https://spack.readthedocs.io/en/latest/packaging_guide.html#git

Reply:
Would `oasis@4.0` describe this version accurately? If so, `version('4.0', ...` would be the way to achieve this.
Right now spack would represent this version of the package with `oasis@OASIS3-MCT_4.0`, which is a bit verbose.

Reply:
Yes I know, it's quite awkward, it's fine to just use 4.0, the rest is frozen anyhow.